tools/fluent/basic_syntax.md

```diff
@@ -104,11 +104,39 @@ emails = { $unreadEmails ->
 
 Note how the `[one]` form in English doesn’t explicitly use the variable, in order to display the word `one` instead of the digit `1`.
 
-In plural messages is always possible to expose the number, even if the reference language doesn’t; the name of the variable is defined at the beginning of the select expression, in this case `$unreadEmails`. The first example in this section can be translated in Italian using `$unreadEmails` in both forms:
+In plural messages is always possible to expose the number, even if the reference language doesn’t; the name of the variable is defined at the beginning of the select expression (in the *selector*), in this case `$unreadEmails`. The first example in this section can be translated in Italian using `$unreadEmails` in both forms:
 
 ```PROPERTIES
 emails = { $unreadEmails ->
         [one] C’è { $unreadEmails } messaggio non letto.
        *[other] Ci sono { $unreadEmails } messaggi non letti.
     }
 ```
+
+## Variants and Terms
+
+As described at the beginning of the document, **terms** are a special type of **messages**: they can be recognized because of the identifier starting with a dash, e.g. `-brand-short-name`, and can only be referenced within other messages. More important, they are allowed to define additional attributes compared to the reference language (typically `en-US`).
+
+While in most cases terms will have a single value, it’s also possible to define multiple **variants**: different values, each one associated to a key, that can be selected when referencing the term ID from another message (or term). Unlike select expressions with a selector (e.g. the `emails` message in the previous section), the value retrieved doesn’t depend on external variables or functions.
```

Comment:
Perhaps show the syntax here right away? I'd also remove the last sentence because it explains a pretty complex part of the design and I'm not sure it's worth distracting the reader with it.

> While in most cases terms will have a single value, it’s also possible to define multiple **variants**: different values, each one associated to a key. Variants represent different forms of the same value. They can be used to define grammatical cases or any other language-specific modifications of the value required by the grammar of the spelling rules. When referencing a term from another message, you can specify which variant to use with the `-term-identifier[variant name]` syntax.

> Consider the following example in English…